### ISSUE TITLE
Kuboo - Archive Mode

### ISSUE THREAD

**Issue by befora** [befora/Kuboo#72]
This project has been great fun but it's time to exit active development. I have gone through two major refactors and put hundreds of hours into this project over a span of around two years. It was a great learning experience considering the many challenges of the Android platform. I'm happy to offer this software to any of you users that enjoy streaming comics for free without any annoying ads, micro-transactions, or invasive data collection.

I am starting to receive propositions for actual paid development work so my focus will be elsewhere. The project is going to enter a read-only Archive Mode. No new features will be implemented but dependencies may be updated. Application crashes are at a minimum and I am happy where the feature sets are at. **Please feel free to fork and do anything you want with the code base.**

Effective March 2019, one month's time from now to clean up any loose ends.

Thank you for all the feedback and bug reports.
Thank you to the open source libraries and communities.
Thank you to Tom for the Ubooquity server software.

-Seth Chhim

**Comment by lbesnard** [comment]
Thanks a lot for all your great efforts ! It's a great software

**Comment by cesalo** [comment]
Yes .. thanks a lot - Amazing work so far, sad news that you won't pursue the server side .. that looked promising.

**Comment by sethchhim** [comment]
@cesalo It was just a test to see how frontend web development and backend server works. Turns out it's all pretty easy, the hard part is support and testing which is time consuming. If anyone is interested just extract the jar, it is not minified so the source can easily be reviewed/copied.
